main.py

```diff
@@ -217,6 +219,51 @@ def _load_switch(self, switch_id, switch_att):
         with self._links_lock:
             self.load_interfaces_tags_values(switch, intf_details)
 
+    def links_activation_consistency_check(self) -> None:
+        """Links activation consistency check.
+
+        Currently, there was a case where a link was down but
+        its interfaces were up. This shouldn't happen in normal circumstances,
+        and consistency will aid these cases if they ever happen while the
+        root case is still being identified and fixed. Either way, consistency
+        will provide an extra guarantee here for links activation.
+
+        https://github.com/kytos-ng/topology/issues/188
+        """
+        with self._links_lock:
+            inconsistent_links = []
+            for link in self.links.values():
+                if not all((
```

Comment:
I'm going to extract this conditional in another func just so if it the predicate changes it becomes easier to extend and test.